src/bundlers/node/nodeJsBinaryDependenciesBundler.ts

```diff
@@ -11,7 +12,11 @@ import packageManager from "../../packageManagers/packageManager.js";
 import { UserError } from "../../errors.js";
 
 export class NodeJsBinaryDependenciesBundler implements BundlerInterface {
-    async #handleBinaryDependencies(dependenciesInfo: Dependency[], tempFolderPath: string) {
+    async #handleBinaryDependencies(
+        dependenciesInfo: Dependency[],
+        tempFolderPath: string,
+        architecture: string,
```

Comment:
Can you make this an enum? This way we ensure no other weird strings can be passed.

I'm thinking something like:
```ts
enum Architecture {
    ARM;
    X86;
}
```

Reply:
Done, thank you for the suggestion, its much better now!